workflow/Snakefile

```diff
@@ -108,3 +108,23 @@ rule generate_min_pvalue_table:
         input_path=ws_path("min_P"),
     shell:
         "python workflow/scripts/create_report_table.py -i {params.input_path} -o {output}"
+
+rule filter_heterogeneity:
+    input:
+        ws_path("output/{seqid}/{seqid}.gwaslab.tsv.gz"),
+    output:
+        ws_path("output/{seqid}/{seqid}_heterogenous_snp.tsv"),
+    conda:
+        "envs/r_environment.yml"
+    resources:
+        runtime=lambda wc, attempt: attempt * 10,
+    params:
+        NEF=3978,
```

Comment:
Better to have this parameter specified in the config/config.yaml file.
Add a new key to the "params" dictionary there, e.g.:
params:
  filter_heterogeneity:
    nef: 3978

then in the rule, you can call it in this way: NEF=config.get("params").get("nef")